docs/releasenotes/index.rst

```diff
@@ -4,7 +4,7 @@ Release Notes
 
 .. note:: If you already have a tutorial environment from a previous release see :ref:`upgrading-vagrant-environment`.
 
-v0.9
+v1.0
```

Comment:
:question: should I be checking that the release notes have all the messages requested by the product and marketing people?